lib/app/modules/timer/views/timer_view.dart

```diff
@@ -145,11 +158,15 @@ class TimerView extends GetView<TimerController> {
                               highlightedTextStyle:
                                   Theme.of(context).textTheme.displayMedium,
                               onTimeChange: (dateTime) {
+                                final hours = dateTime.hour;
+                                final minutes = dateTime.minute;
+                                final seconds = dateTime.second;
+                                timeLabel='Timer Duration - $hours hrs : $minutes min : $seconds sec';
```

Comment:
You'll need to add the new translations as well for texts.

Reply:
Sure ! On it